### ISSUE TITLE
Auto-skip non-image files

### ISSUE THREAD

**Issue by mvysny** [mvysny/photocloud-frame-slideshow#11]
Make PhotoCloud auto-skip non-image files. Just a simple extension-based filter is enough. The filter should already be employed, but apparently it's not effective in all places:
```
Non-fatal Exception: java.io.IOException: Failed to detect dimensions of /storage/emulated/0/Android/data/sk.baka.photoframe/files/tmp_download/tmp_download/aaa83246165.bash_logout
       at sk.baka.photoframe.fsimpl.cache.CacheBitmapTransformer.load(CacheBitmapTransformer.kt:91)
       at sk.baka.photoframe.frame.PhotoDownloader$DownloadImageRunnable.run(PhotoDownloader.java:241)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
```

**Comment by mvysny** [comment]
Related Fabric issue: https://fabric.io/baka/android/apps/sk.baka.photoframe/issues/590ccde2be077a4dcc57d2a7

**Comment by mvysny** [comment]
Another related issue: https://fabric.io/baka/android/apps/sk.baka.photoframe/issues/595f2403be077a4dcc2af8e0
But the image filter is there and is present! It's only not present when one explicitly chooses AVI from the file browser - that must be it! I'll try to disallow opening of non-image files from the Browser UI, we'll see whether it helped or not.

**Comment by mvysny** [comment]
Fixed in 1.9.1
